torch/nn/quantized/functional.py

```diff
@@ -472,8 +472,8 @@ def threshold(input, threshold, value):
         raise ValueError("Input to 'value' must be specified!")
     return torch._ops.ops.quantized.threshold(input, threshold, value)
 
-def elu(input, alpha=1., inplace=False, scale=None, zero_point=None):
-    # type: (Tensor, Optional[float], bool, Optional[float], Optional[int]) -> Tensor
+def elu(input, scale, zero_point, alpha=1.):
```

Comment:
why the functional has less arguments than the quantized op? missing `scale` and `input_scale`

Reply:
I agree, it's a bit confusing.  The quantized python op is following the floating point python op (https://pytorch.org/docs/stable/_modules/torch/nn/modules/activation.html#ELU).  The 'scale' and 'input_scale' arguments seen in c++ for ELU for both fp and quant are not related to quantization, and also not exposed to Python.  I believe they are there because the kernel is reused with other functions, which do use these params.

Reply:
I see